client/src/components/add-job-post-component/application-form-fields/index.jsx

```diff
@@ -0,0 +1,143 @@
+import UploadFileIcon from "@mui/icons-material/UploadFile";
+import {
+  Box,
+  Container,
+  FormControl,
+  IconButton,
+  InputLabel,
+  Select,
+  TextField,
+  FormHelperText,
+  Typography,
+} from "@mui/material";
+import { ButtonL, JobLeadContainer, H3 } from "../index.styles";
+import styles from "./index.module.css";
+
+function AddApplicationFields() {
+  return (
+    <>
+      <JobLeadContainer>
+        <div
+          style={{
+            display: "flex",
+            justifyContent: "space-between",
+            alignItems: "center",
+          }}
+        >
+          <H3>Application form fields</H3>
+        </div>
+
+        {/* Name Field */}
+        <TextField
+          disabled
+          fullWidth
+          className={styles.dotted}
+          sx={{ m: 1, width: "47%" }}
+          label="Name"
+          helperText="*Required"
+          required
+          InputLabelProps={{ required: false }}
+        />
+
+        {/* Email Field */}
+        <TextField
+          disabled
+          fullWidth
+          className={styles.dotted}
+          sx={{ m: 1, width: "47%" }}
+          label="Email Address"
+          helperText="*Required"
+          required
+          InputLabelProps={{ required: false }}
+        />
+
+        {/* Phone Field */}
+        <TextField
+          disabled
+          fullWidth
+          className={styles.dotted}
+          sx={{ m: 1, width: "47%" }}
+          label="Phone"
+          helperText="*Required"
+          required
+          InputLabelProps={{ required: false }}
+        />
+
+        {/* Postal Code Field */}
+        <TextField
+          disabled
+          fullWidth
+          className={styles.dotted}
+          sx={{ m: 1, width: "47%" }}
+          label="Postal Code"
+          helperText="*Required"
+          requiredf
+          InputLabelProps={{ required: false }}
+        />
+
+        {/* Status In Canada Field */}
+        <FormControl disabled fullWidth sx={{ m: 1, width: "96%" }}>
+          <InputLabel>Status In Canada</InputLabel>
+          <Select className={styles.dotted} required />
+          <FormHelperText>*Required</FormHelperText>
+        </FormControl>
+
+        {/* Cover Letter Field */}
```

Comment:
I don't know if there is a cover letter field in the model file for Job Applications - Might have to check that!

Although, it could possibly be included in the 'custom_responses' attribute for the Job Application Model but I'm not too sure though.